components/PromoSection.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+	<section class="mx-auto mb-40 mt-20 max-w-3xl space-y-8 px-4">
+		<SectionTitle
+			title="Das Open Source Invoice Tool für Freelancer und kleine Unternehmen"
+			align="center"
+			color-class="text-gray-700"
+		/>
+
+		<p class="text-center text-xl text-gray-600">
+			Unser Rechnungsprogramm ermöglicht es Ihnen, schnell und effektiv
+			Rechnungen zu generieren und sofort an Kunden zu senden. Dies beschleunigt
+			den Zahlungseingang und verbessert Ihren Cashflow.
+		</p>
+
+		<div
+			class="flex flex-col items-center justify-center space-y-8 rounded-5xl border bg-white p-8 shadow-xl"
+		>
+			<img
+				alt="Das Open Source Invoice Tool"
+				src="https://placehold.co/400x300"
+				class="mt-8 rounded-[3rem] border-4 border-[#95BAFC] border-opacity-20"
+			>
+			<h3 class="text-left text-4xl font-bold text-blue-500">
+				Das Open Source Invoice Tool
+			</h3>
+			<p class="text-2xl font-semibold text-gray-600">
+				Ideal für Freelancer und kleine Unternehmen die eine anpassbare und
+			</p>
+		</div>
```

Comment:
Thought: This is another type of content card. Maybe we can combine all type into one Abstract ContentCard Type that we can assign a type to.